package.json

```diff
@@ -50,7 +50,8 @@
     "jest-diff": "18.1.0",
     "jest-snapshot": "18.1.0",
     "react-test-renderer": "15.4.1",
-    "strip-ansi": "3.0.1"
+    "strip-ansi": "3.0.1",
+    "whatwg-fetch": "2.0.3"
```

Comment:
I’d suggest https://github.com/developit/unfetch as a smaller alternative since you’re going to send this polyfill to all users.